helix-view/src/editor.rs

```diff
@@ -1229,6 +1232,10 @@ impl Editor {
             self.mode = Mode::Normal;
             self.ensure_cursor_in_view(view_id);
         }
+
+        let view = self.tree.get_mut(view_id);
```

Comment:
```suggestion
        let view = view!(self, view_id);
```

And for the one below

Reply:
done